### PR TITLE
feat: consolidate voter's VOC

### LINKED ISSUE #21: allow flexible number of inputs and also input consolidation for voiceCredits.

### PR DESCRIPTION
**Pull request by troggy**
Everytime voter executed spendie, use 1 largest and 3 smallest VOC outputs, so they
squashed together in a single change output.

Resolves #21

### CODE REVIEW

**Review by MaxStalker — APPROVED** [review]

**Review by MaxStalker — APPROVED** [review]

**Comment by troggy** [comment]
@MaxStalker @johannbarbie updated this one. I think there was a bug with number of inputs signed. Please review again

**Comment by MaxStalker** [comment]
> @MaxStalker @johannbarbie updated this one. I think there was a bug with number of inputs signed. Please review again

You pass it as array, I think we are good then. Since we sign based on number of personal inputs (or at least that how it was).

**Comment by troggy** [comment]
Probably resolves #69 and #72 as well

**Review by MaxStalker — APPROVED** [review]
PR looks solid. Now we need to test it :)

File: src/volt/components/VoteControls/index.js [hunk 1]
```diff
@@ -121,18 +126,51 @@ class VoteControls extends Component {
     };
   }
 
-  async getVoteCredits(address) {
+
+  async getMyVoteCredits() {
```
Comment (johba37) [inline src/volt/components/VoteControls/index.js:130]: it is `voiceCredits`, not `vote` 😎

**Review by johba37 — APPROVED** [review]
only syntax

**Pull request merged**